integration_test/install_test.go

```diff
@@ -105,6 +107,20 @@ func TestKrewInstall_Manifest(t *testing.T) {
 	test.AssertExecutableInPATH("kubectl-" + validPlugin)
 }
 
+func TestKrewInstall_ManifestURL(t *testing.T) {
+	skipShort(t)
+
+	test, cleanup := NewTest(t)
+	defer cleanup()
+	srv, close := localTestServer()
+	defer close()
```

Comment:
I'm not exactly sure what it is shadowing, but gocritic is unhappy about calling it `close`.

Reply:
shadows close builtin method on go channels :)

Lets use cleanup.

Reply:
:bulb: `cleanup` sounds good.

Reply:
fixed.